.github/workflows/playwright-examples.yml

```diff
@@ -49,6 +49,10 @@ jobs:
       - name: Run Playwright tests
         run: 'yarn e2e'
         working-directory: apps/examples
+        env:
+          PERFORMANCE_ANALYTICS_ENABLED: true
```

Comment:
We probably want to remove these, otherwise we'll run perf tests with this workflow as well?